cpp/core/CMakeLists.txt

```diff
@@ -264,6 +264,17 @@ if(ENABLE_HBM)
   endif()
 endif()
 
+if(ENABLE_QAT)
+  find_library(QATZIP_LIBRARY qatzip)
+  if(MEMKIND_LIBRARY STREQUAL "QATZIP_LIBRARY-NOTFOUND")
+    message(FATAL_ERROR "QATZip Library Not Found")
+  endif()
+  message(STATUS "Found QATZip Library: ${QATZIP_LIBRARY}")
+  target_sources(spark_columnar_jni PRIVATE utils/qat_util.cc)
+  target_link_libraries(spark_columnar_jni PUBLIC ${QATZIP_LIBRARY})
+  add_definitions(-DGLUTEN_ENABLE_QAT)
+endif()
+
```

Comment:
Does the script install the qat library automatically? If not, we may need to add the way to pre-install qat.

Reply:
We need to both build and install QAT driver and QATzip. For QAT driver, we've only tested with the internal version. The public version can be downloaded [here](https://www.intel.com/content/www/us/en/download/19678/intel-quickassist-technology-intel-qat-driver-for-windows-for-intel-server-boards-and-systems-based-on-intel-62x-chipset.html), but the version is old.